pages/apis/rest_api/pipelines.md

```diff
@@ -378,6 +378,13 @@ Optional [request body properties](/docs/api#request-body-properties):
       <p><em>Example:</em> <code>"!main"</code><br><em>Default:</em> <code>null</code></p>
     </td>
   </tr>
+  <tr>
+    <th><code>slug</code></th>
+    <td>
+      <p>A custom identifier for the pipeline. If provided, this slug will be used as the pipeline's URL path instead of automatically converting the pipeline name. If the value is `nil`, the pipeline name will be used to generate the slug.</p>
```

Comment:
I know it's an optional parameter, so I feel like saying "If the value is `nil` is a bit redundant. But I thought I might be important to clarify that it will default to the original behaviour if someone provided `"slug": "nil"`

Reply:
*This comment is not outdated. I just fixed a formatting error